src/world/FieldComputations.cpp

```diff
@@ -185,125 +94,113 @@ std::shared_ptr<Vector2> FieldComputations::lineIntersectionWithDefenceArea(cons
 
     if (intersections.size() == 1) {
         return std::make_shared<Vector2>(intersections.at(0));
```

Comment:
```suggestion
        return intersections[0];
```

Reply:
I am afraid to change it. I know at a lot of places in the code they did this to prevent the variable from being forgotten which causes strange random runtime errors.